_overviews/scala3-book/methods-most.md

```diff
@@ -233,7 +234,7 @@ Here’s a method named `isTruthy` that implements the Perl definitions of `true
 
 ```scala
 def isTruthy(a: Any) =
-  if a == 0 || a == "" then
+  if a == 0 || a == "" || a == false then
```

Comment:
Wow, thanks, this is a great catch! No telling how many times I’ve written that wrong.